### PR TITLE
Allow to sort tables

### PR DESCRIPTION
**Pull request by snukky**
Added sorting to tables in leaderboard and team pages, based on jQuery tablesorter 2.0: https://mottie.github.io/tablesorter/docs/example-option-theme-bootstrap-v4.html

Changes proposed in the pull request:
- Set up `assets/` directory for CSS, JS, images that are not specific to an app.
- Added tablesorter static files.
- Enabled sorting on the leaderboard and team pages.

This is added on top of #16 .

@AppraiseDev/core-team

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 - Running multiple competitions at the same time with own deadlines
+- Sorting result tables with jQuery tablesorter 2.0
```
Comment (cfedermann) [inline CHANGELOG.md:11]: @snukky: specifically mention that we opt for `MIT` license here?

File: leaderboard/templates/leaderboard/competition.html [hunk 2]
```diff
@@ -92,12 +96,12 @@ <h2>Leaderboard of {{competition.name}}</h2>
...
         <div class="table-responsive">
           <table class="table table-striped table-sm">
-            <thead>
+            <thead class="thead-dark">
               <tr>
                 <th>#</th>
                 <th>Name</th>
-                <th>SacreBLEU score</th>
-                <th>chrF score</th>
+                <th>BLEU</th>
```
Comment (cfedermann) [inline leaderboard/templates/leaderboard/competition.html:103]: @snukky: why drop `SacreBLEU` and refer to `BLEU` only?

File: ocelot/settings.py [hunk 3]
```diff
@@ -166,6 +166,11 @@
 STATIC_URL = '/static/'
 STATIC_ROOT = os.path.join(BASE_DIR, 'static')
```
Comment (cfedermann) [inline ocelot/settings.py:167]: @snukky: again, should we switch to `pathlib.Path`?

**Review by cfedermann — CHANGES_REQUESTED** [review]
@snukky: let's talk about this in 11 minutes. Thanks!

File: CHANGELOG.md (hunk 1 above)
Comment (snukky) [reply CHANGELOG.md:11]: Yes, it's double licensed, so this totally makes sense.

File: leaderboard/templates/leaderboard/competition.html (hunk 2 above)
Comment (snukky) [reply leaderboard/templates/leaderboard/competition.html:103]: Yes, for brevity. Table sorting adds extra elements to `<thead>`s.

File: ocelot/settings.py (hunk 3 above)
Comment (snukky) [reply ocelot/settings.py:167]: Sure. Will do.

**Review by cfedermann — APPROVED** [review]
Good to go.

File: ocelot/settings.py (hunk 3 above)
Comment (snukky) [reply ocelot/settings.py:167]: As mentioned in another pull request, I will open a new PR for this.

**Pull request merged**